library/src/main/kotlin/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -196,7 +210,59 @@ public class ChuckerInterceptor private constructor(
             }
 
         /**
-         * Creates a new [ChuckerInterceptor] instance with values defined in this builder.
+         * Sets a list of [Regex] to skip paths. When any of the [Regex] matches a
+         * request path, the request will be skipped. Include [RegexOption] where
+         * necessary.
+         *
+         * ```
+         * ".*(jpg|jpeg|png|gif|webp|svg|bmp|ico)$".toRegex(), // Ignore all image requests
+         *  ".*iGnOrE.*"toRegex(RegexOption.IGNORE_CASE), // Case insensitive
+         *  ".*path/to/skip.*".toRegex(),
+         *  ".*path/ends/with/dev$".toRegex(),
+         * ```
+         */
+        public fun skipPaths(skipPaths: Regex): Builder =
+            apply {
+                this.skipPathsRegex.add(skipPaths)
+            }
+
+        /**
+         * Sets a list of [String] to skip domains. Domain names are evaluated in
+         * lowercase format. When any of the [String] matches a request domain,
+         * the request will be skipped.
+         * ```
+         * example.com, subdomain.example.com, exam-ple.com, eXaMpLe.CoM, example.co.uk
+         * ```
+         * @throws PatternSyntaxException if the domain pattern is invalid.
+         */
+        @Throws(PatternSyntaxException::class)
+        public fun skipDomains(vararg skipDomains: String): Builder =
+            apply {
+                skipDomains.forEach { domain ->
+                    this.skipDomainRegex.add("^${domain}\$".toRegex(RegexOption.IGNORE_CASE))
```

Comment:
`mydomain.*.com` will match `mydomain-anything.com`; This is not expected behaviour. But, this is a shortcut to avoid two loops.


Should this be updated?